frontend/src/Components/RevealButton/RevealButton.tsx

```diff
@@ -32,7 +32,12 @@ export const RevealButton = connectToWebSocket(
     const { missing, voted } = getNumberOfVotes(votes);
 
     return (
-      <button class={classes.revealButton} onClick={revealVotes} disabled={!connected || !voted}>
+      <button
+        data-testid="reveal-button"
```

Comment:
Can we use a meaningful but static aria label, such as `Reveal votes`, instead?

Reply:
Definitely. I actually did this in the next PR #112 but I will move the change to this PR instead